scheduler/src/cook/rest/api.clj

```diff
@@ -1968,10 +1969,14 @@
   ::jobs and ::groups, which specify the jobs and job groups."
   [conn {:keys [::groups ::job-pool-name-maps] :as ctx}]
   (try
-    (log/info "Submitting jobs through raw api:"
-              (map (fn [job-pool-name-map]
-                     (update job-pool-name-map :job #(dissoc % :command)))
-                   job-pool-name-maps))
+    (let [jobs (map (fn [job-pool-name-map]
+                      (update job-pool-name-map :job #(dissoc % :command)))
+                    job-pool-name-maps)]
+      (log/info "Submitting jobs through raw api:" jobs)
+      (doseq [{{:keys [uuid, user]} :job} jobs]
+        (passport/log-event {:job-uuid (str uuid)
+                                      :user user
+                                      :event-type passport/api-job-submission})))
```

Comment:
Indentation seems off on these two lines

Reply:
Are you sure? What do you think is wrong?

Reply:
Ah got this way after changing `log-passport-event` to `log-event`, I'll fix it.